tools/ci/common.sh

```diff
@@ -21,6 +21,12 @@ RENAISSANCE_GIT_VERSION=$(git describe --tags --always --dirty=-SNAPSHOT || echo
 # Strip leading 'v' from the git-produced version
 RENAISSANCE_VERSION=${RENAISSANCE_GIT_VERSION#v}
 
+# Try to guess JVM version
+RENAISSANCE_JVM_MAJOR_VERSION="$( java -XshowSettings:properties -version 2>&1 \
+    | sed -n -e 's#^[ \t]*java.version[ ]*=[ ]\(.*\)#\1#p' \
```

Comment:
I would write the `sed` expression as follows:
```
sed -n '/^[[:blank:]]*java[.]version[[:blank:]]*=/ { s/.*=[[:blank:]]*\([^[:space:]]\+\)/\1/; s/^1[.]//; s/^\([^.]\+\).*/\1/; p }'
```
but it's your call. I'm a bit uncomfortable with the `#` characters in the `sed` expression (my brain sees comments).

Also, is it necessary to use the `-X` option? The following works with just the `-version` output on JDK 8, 11, 16, and Hotspot 15. On OpenJ9 11, it gets the opendjk version from the first line, which is probably fine too:
```
sed -n '/version[[:blank:]]\+/ { s/.*version[[:blank:]]\+"\([^[:space:]"]\+\)/\1/; s/^1[.]//; s/^\([^.]\+\).*/\1/; p }'
```

If we really want to use the `-X` option, why not ask for the `java.specification.version`? The `sed` expression only needs to strip `1.` from the `1.8` version (you could even keep it there).
```
 sed -n '/^[[:blank:]]*java[.]specification[.]version[[:blank:]]*=/ { s/.*=[[:blank:]]*\([^[:space:]]\+\)/\1/; s/^1[.]//; p }'
```
This worked on OpenJDK 8, 11, 16, Hotspot 15, and OpenJ9 11.

Reply:
Okay, using `java -version` (hopefully they will not unify it with `--version` that has slightly different output) and used `/` as delimiter in `sed` ;-) [tested on JDK 8 to 16 including OpenJ9].